pkg/benchmark/benchmark.go

```diff
@@ -129,26 +183,50 @@ func Run(runs int, profile string) (AggregatedResultsMatrix, error) {
 	}
 
 	for _, method := range BenchMethods {
-		if err := method.startMinikube(profile); err != nil {
-			log.Printf("failed to start %s: %v", method.Name, err)
-			continue
+		skipMethod := false
+		if _, ok := config.BenchMethods[method.Name]; !ok {
+			skipMethod = true
 		}
 
-		for _, mode := range modes {
+		if !skipMethod {
```

Comment:
If you're skipping this method, why don't we just `continue`?

Reply:
Because even if this method should be skipped, we still need to generate an empty slice in the result map for each image and each interactive-method. So the loop below still should be reused and be executed

Reply:
I see, thanks for the explanation